sdcm/sct_config.py

```diff
@@ -261,6 +261,10 @@ class SCTConfiguration(dict):
                   Time in minutes, Time of execution for stress commands from stress_cmd parameters
                   and is used in test duration calculation
              """),
+        dict(name="stress_target", env="SCT_STRESS_TARGET", type=int,
+             help="""
+                  Number of operations per second to achieve in stress commands.
```

Comment:
in you case it effects all of the ycsb calls  ? can you elaborate  a bit more ? 

if it's currently only relevant to perf test, and for ycsb/alternator based on 
I would name it as such `alternator_perf_stress_target` 

otherwise people can assume they can put a number here, and it would rate limit their cassandra-stress command, while it's not gonna do that...